--- ngx_cache_purge_module.c    2014-12-23 18:40:21.000000000 +0000
+++ ngx_cache_purge_module.c.new        2019-08-09 10:29:11.910407977 +0000
@@ -191,7 +191,6 @@

 static char ngx_http_cache_purge_success_page_tail[] =
 CRLF "</center>" CRLF
-"<hr><center>" NGINX_VER "</center>" CRLF
 "</body>" CRLF
 "</html>" CRLF
 ;
@@ -1261,8 +1260,6 @@

     len = sizeof(ngx_http_cache_purge_success_page_top) - 1
           + sizeof(ngx_http_cache_purge_success_page_tail) - 1
-          + sizeof("<br>Key : ") - 1 + sizeof(CRLF "<br>Path: ") - 1
-          + key[0].len + r->cache->file.name.len;

     r->headers_out.content_type.len = sizeof("text/html") - 1;
     r->headers_out.content_type.data = (u_char *) "text/html";
@@ -1286,12 +1283,6 @@

     b->last = ngx_cpymem(b->last, ngx_http_cache_purge_success_page_top,
                          sizeof(ngx_http_cache_purge_success_page_top) - 1);
-    b->last = ngx_cpymem(b->last, "<br>Key : ", sizeof("<br>Key : ") - 1);
-    b->last = ngx_cpymem(b->last, key[0].data, key[0].len);
-    b->last = ngx_cpymem(b->last, CRLF "<br>Path: ",
-                         sizeof(CRLF "<br>Path: ") - 1);
-    b->last = ngx_cpymem(b->last, r->cache->file.name.data,
-                         r->cache->file.name.len);
     b->last = ngx_cpymem(b->last, ngx_http_cache_purge_success_page_tail,
                          sizeof(ngx_http_cache_purge_success_page_tail) - 1);
     b->last_buf = 1;
