--- ngx_cache_purge-2.3/ngx_cache_purge_module.c	2014-12-23 18:40:21.000000000 +0000
+++ ngx_cache_purge_module.c	2019-08-10 16:49:45.473331212 +0000
@@ -191,7 +191,6 @@

 static char ngx_http_cache_purge_success_page_tail[] =
 CRLF "</center>" CRLF
-"<hr><center>" NGINX_VER "</center>" CRLF
 "</body>" CRLF
 "</html>" CRLF
 ;
@@ -1253,16 +1252,11 @@
 {
     ngx_chain_t   out;
     ngx_buf_t    *b;
-    ngx_str_t    *key;
     ngx_int_t     rc;
     size_t        len;

-    key = r->cache->keys.elts;
-
     len = sizeof(ngx_http_cache_purge_success_page_top) - 1
-          + sizeof(ngx_http_cache_purge_success_page_tail) - 1
-          + sizeof("<br>Key : ") - 1 + sizeof(CRLF "<br>Path: ") - 1
-          + key[0].len + r->cache->file.name.len;
+          + sizeof(ngx_http_cache_purge_success_page_tail) - 1;

     r->headers_out.content_type.len = sizeof("text/html") - 1;
     r->headers_out.content_type.data = (u_char *) "text/html";
@@ -1286,12 +1280,6 @@

     b->last = ngx_cpymem(b->last, ngx_http_cache_purge_success_page_top,
                          sizeof(ngx_http_cache_purge_success_page_top) - 1);
-    b->last = ngx_cpymem(b->last, "<br>Key : ", sizeof("<br>Key : ") - 1);
-    b->last = ngx_cpymem(b->last, key[0].data, key[0].len);
-    b->last = ngx_cpymem(b->last, CRLF "<br>Path: ",
-                         sizeof(CRLF "<br>Path: ") - 1);
-    b->last = ngx_cpymem(b->last, r->cache->file.name.data,
-                         r->cache->file.name.len);
     b->last = ngx_cpymem(b->last, ngx_http_cache_purge_success_page_tail,
                          sizeof(ngx_http_cache_purge_success_page_tail) - 1);
     b->last_buf = 1;
